gui/filehandling.py

```diff
@@ -798,7 +799,7 @@ def save_cb(self, action):
         if not self.filename:
             self.save_as_cb(action)
         else:
-            self.save_file(self.filename)
+            self.save_file(self.filename, alpha=True)  # defaults to transparent background
```

Comment:
I'm getting an error that probably caused by this.

Steps to Reproduce:
1) Open with a Blank Canvas, and scribble on it
2) Save as Ora
3) Save again with File>Save

**Traceback Dump:**
```python
Mypaint version: 1.3.0-alpha+git.9f0e6f8
System information: Linux-4.2.0-34-generic-x86_64-with-Ubuntu-15.10-wily
Using: Python 2.7.10, GTK 3.16.7, GdkPixbuf 2.32.1, Cairo 1.14.4
Traceback (most recent call last):
  File "/home/albert/Public/Github/mypaint/gui/filehandling.py", line 802, save_cb(self=<gui.filehandling.FileHandler object>, action=<Action object at 0x7f90af588320 (GtkAction at 0x1aa51f0)>)
            else:
                self.save_file(self.filename, alpha=True)  # defaults to transparent background
  variables: {'alpha': (None, []), 'self.save_file': ('local', <bound method FileHandler.save_file of <gui.filehandling.FileHandler object at 0x7f90af57a550>>), 'True': ('builtin', True), 'self.filename': ('local', u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora')}
  File "/home/albert/Public/Github/mypaint/gui/drawwindow.py", line 85, wrapper(self=<gui.filehandling.FileHandler object>, *args=(u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora',), **kwargs={'alpha': True})
            try:
                func(self, *args, **kwargs)
                # gtk main loop may be called in here...
  variables: {'self': ('local', <gui.filehandling.FileHandler object at 0x7f90af57a550>), 'args': ('local', (u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora',)), 'func': ('local', <function save_file at 0x7f90af60a7d0>), 'kwargs': ('local', {'alpha': True})}
  File "/home/albert/Public/Github/mypaint/gui/filehandling.py", line 555, save_file(self=<gui.filehandling.FileHandler object>, filename=u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora', export=False, **options={'alpha': True})
                statusmsg=True,
                **options
            )
  variables: {'options': ('local', {'alpha': True})}
  File "/home/albert/Public/Github/mypaint/gui/filehandling.py", line 636, _save_doc_to_file(self=<gui.filehandling.FileHandler object>, filename=u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora', doc=<gui.document.Document object>, export=False, statusmsg=True, **options={'alpha': True, 'save_srgb_chunks': True})
                    feedback_cb=self.gtk_main_tick,
                    **options
                )
  variables: {'options': ('local', {'alpha': True, 'save_srgb_chunks': True})}
  File "/home/albert/Public/Github/mypaint/lib/document.py", line 1212, save(self=<Document nlayers=32 bbox=Rect(0, 0, 2176, 3456) paintonly=False>, filename=u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora', **kwargs={'alpha': True, 'feedback_cb': <function gtk_main_tick>, 'save_srgb_chunks': True})
            try:
                result = save(filename, **kwargs)
            except GObject.GError as e:
  variables: {'kwargs': ('local', {'feedback_cb': <function gtk_main_tick at 0x7f90af60a5f0>, 'alpha': True, 'save_srgb_chunks': True}), 'save': ('local', <bound method Document.save_ora of <Document nlayers=32 bbox=Rect(0, 0, 2176, 3456) paintonly=False>>), 'result': ('local', None), 'filename': ('local', u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora')}
  File "/home/albert/Public/Github/mypaint/lib/fileutils.py", line 118, _wrapped_save_method(self=<Document nlayers=32 bbox=Rect(0, 0, 2176, 3456) paintonly=False>, filename=u'/home/albert/Artwork/Current Projects/Linux Showdown/Source Files/2016-0330-Team_Noah02.ora', *args=(), **kwds={'alpha': True, 'feedback_cb': <function gtk_main_tick>, 'save_srgb_chunks': True})
                    logger.error("cleanup: failed to remove temp path too")
                raise ex
            if not os.path.exists(temp_path):
  variables: {'ex': ('local', TypeError("save_as_png() got multiple values for keyword argument 'alpha'",))}
TypeError: save_as_png() got multiple values for keyword argument 'alpha'
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
```
I did remove the change you made in that line and that removed the error. However MyPaint freezes up randomly when I try to save.